### PR TITLE
Warnings of theano and pymc3 has been deleted

### PR DESCRIPTION
**Pull request by samO2I1**
#5312 
1.I have deleted the warnings of importing theano and pymc3

2.This PR is the solution to the issue[#5312](https://github.com/pymc-devs/pymc/issues/5312)

### CODE REVIEW

File: pymc/__init__.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 import logging
 import multiprocessing as mp
 import platform
-
+import theano
```
Comment (twiecki) [inline pymc/__init__.py]:
```suggestion
```

File: pymc/__init__.py [hunk 2]
```diff
@@ -18,7 +18,8 @@
...
+import pymc3
```
Comment (twiecki) [inline pymc/__init__.py]:
```suggestion
```

**Review by michaelosthege — APPROVED** [review]

**Comment by samO2I1** [comment]
Dear Sir,
The pull request has been approved . What needs to be done next??
Thanks & Regards

**Comment by twiecki** [comment]
Thanks for the contribution @samO2I1!

**Pull request merged**